core-plugins/src/main/java/co/cask/hydrator/plugin/common/FileSetUtil.java

```diff
@@ -47,12 +49,13 @@
    * Parquet input and output format, and also configured for Explore to use Parquet. The schema is added
    * to the file set properties in all the different required ways:
    * <ul>
-   *   <li>As a top-level dataset property;</li>
-   *   <li>As the schema for the input and output format;</li>
-   *   <li>As the schema of the Hive table.</li>
+   * <li>As a top-level dataset property;</li>
+   * <li>As the schema for the input and output format;</li>
+   * <li>As the schema of the Hive table.</li>
    * </ul>
+   *
    * @param configuredSchema the original schema configured for the table
-   * @param properties a builder for the file set properties
+   * @param properties       a builder for the file set properties
```

Comment:
The extra tab is to start the description of properties and configured schema from same position. Something we are following in other javadocs through out the project